pkg/auth/deps.go

```diff
@@ -74,8 +74,24 @@ func ProvideLoggingRequestID(r *http.Request) logging.RequestID {
 	return logging.RequestID(r.Header.Get(corehttp.HeaderRequestID))
 }
 
-func ProvideTenantConfig(ctx context.Context) *config.TenantConfiguration {
-	return config.GetTenantConfig(ctx)
+func ProvideTenantConfig(ctx context.Context, m DependencyMap) *config.TenantConfiguration {
+	// populate default
+	tc := config.GetTenantConfig(ctx)
+	appConfig := tc.AppConfig
```

Comment:
`AppConfig` is a pointer, so we'll need to copy it to avoid mutating the configuration.

Reply:
Updated:)